provider/netperf_test.py

```diff
@@ -57,25 +57,27 @@ def netperf_stress(test, params, vm):
                           params.get_numeric("netperf_para_sessions"),
                           params.get("netperf_cmd_prefix", ""),
                           package_sizes=params.get("netperf_sizes"))
-        if utils_misc.wait_for(n_client.is_netperf_running, 10, 0, 1,
+        if utils_misc.wait_for(n_client.is_netperf_running, 10, 0, 0.1,
```

Comment:
Hi @PaulYuuu According to the results of the test log, it will only print a `Wait netperf test start`, change this to `0.1`, the purpose is to shorten the time to determine whether netperf is started, so that we can quickly jump out of the `utils_misc.wait_for` function. Make the subsequent test more accurate in time judgment.